### DISCUSSION TITLE
使用svg渲染时，图例翻至第2页后无法回到第1页

### DISCUSSION THREAD

**Discussion by imaverickk** [antvis/G2#4989] [Q&A] [answered]
重现方法：
https://g2.antv.vision/zh/examples/pie/basic#pie-spider-label
此示例中添加 renderer: 'svg' 使用svg渲染，缩小视图直到图例被分页，翻至第2页后无法回到第1页。

**Comment by hustcc** [comment, marked as answer]
感谢反馈，在 4.x 版本中，SVG renderer 还存在一些问题，建议还是使用 canvas 渲染。

这个问题我们需要在 G2 新版本中去处理。
